### DISCUSSION TITLE
Event button now grey.  Will log event, but will not prompt me for event type

### DISCUSSION THREAD

**Discussion by undefeaterpeter** [artisan-roaster-scope/artisan#1671] [Q&A]
A few weeks ago the event button changed to always being greyed out and will not prompt me for event type and value.  I've been through settings quite a bit but not seeing any issue.  Hoping not user error with a settings change i don't remember lol

Anybody else with this issue?

<img width="1417" alt="Screenshot 2024-08-15 at 5 57 58 PM" src="https://github.com/user-attachments/assets/d7b6eb0a-bbb7-4bb7-ad78-aa044fdd27f7">

<img width="1417" alt="Screenshot 2024-08-15 at 5 58 30 PM" src="https://github.com/user-attachments/assets/089a3ef8-7e42-4b35-8a3f-ef44df4fa0cd">

**Comment by poundy** [comment]
You're running Artisan 3.0.0. Do you think the change you noticed coincided with upgrading to that version?

**Reply by undefeaterpeter** [reply]
don’t think so. the issue happened before i upgraded, i upgraded hoping it would solve it

**Reply by poundy** [reply]
are you suggesting the issue "just happened" on the same version of the software? Like, one day it worked, next it didn't ?  

I don't know what settings may affect this, but to me this is going to come down to something in your configuration changed.

**Comment by MAKOMO** [comment]
The default button color changed some versions ago (maybe in v2) and the button action never prompted for the event value and type. Instead, open the "Events Editor" (menu View) and change the value and type of the created event afterwards.
